# try:
#     numerator=10
#     denomerator=0
#     r=numerator%denomerator
#     print(r)
# except:
#     print("erroe:denominator cannot be 0")

# try:
#     even_n=[2,4,6,8]
#     print(even_n[5])
# except ZeroDivisionError:
#     print("denominator cannot be 0.")
# except IndexError:
#     print("index out of bound.")

# try:
#     print(x)
# except NameError:
#     print("varibe x is not defined")
# except:
#     print("something else went wrong")

# try:
#     print("hello")
# except:
#     print("something went wrong")
# finally:
#     print("the 'try except' is finished")

